src/grid/CallLayout.ts

```diff
@@ -110,38 +109,33 @@ export function arrangeTiles(
   // use of screen space for n tiles without making those tiles too small or
   // too cropped (having an extreme aspect ratio)
   const gap = width < 800 ? 16 : 20;
-  const tileMinWidth = width < 500 ? 150 : 180;
+  const area = width * minHeight;
+  // Magic numbers that make tiles scale up nicely as the window gets larger
+  const tileArea = Math.pow(Math.sqrt(area) / 8 + 125, 2);
+  const tilesPerPage = Math.min(tileCount, area / tileArea);
 
-  let columns = Math.min(
+  const columns = Math.min(
     // Don't create more columns than we have items for
-    tileCount,
+    tilesPerPage,
     // The ideal number of columns is given by a packing of equally-sized
     // squares into a grid.
     // width / column = height / row.
     // columns * rows = number of squares.
     // ∴ columns = sqrt(width / height * number of squares).
     // Except we actually want 16:9-ish tiles rather than squares, so we
     // divide the width-to-height ratio by the target aspect ratio.
```

Comment:
```suggestion
    // The relation between the tile aspect ratio and the available area aspect ratio
    // defines the optimal ratio between columns and rows.
    // areaAspectRatio / tileAspectRatio = columns / rows
    // substitute: rows = tilesPerPage / columns
    // columns^2 = areaAspectRatio / tileAspectRatio  * tilesPerPage
```
This might be an equally simple approach without the step of inserting `tileMinAspectRatio` at the correct location out of intuition.